### PR TITLE
[search] Pastry/Cake categories.

### PR DESCRIPTION
**Pull request by vng**

### CODE REVIEW

File: data/categories.txt [hunk 1]
```diff
@@ -1104,7 +1104,7 @@ eu:3Okindegia
 sv:3Bageri
 th:3เบเกอรี่
 tr:3Fırın
-uk:3Пекарня
+uk:3Пекарня|Торт|3Торти
```
Comment (strump) [inline data/categories.txt:1107]: Other Ukrainian synonims to "cake": `пиріг` (pie), `кекс` (cupcake), `тістечко` (pastry)

Comment (biodranik) [reply data/categories.txt:1107]: Here should be terms that people likely type in the search to find what they're looking for.

File: data/categories.txt [hunk 2]
```diff
@@ -2151,10 +2151,10 @@ sk:Obchod s výpočtovou technikou
 fa:فروشگاه کامپیوتر
 
 shop-confectionery|craft-confectionery|@category_food|@shop
-en:4Sweets|4confectionery|candies|candy|U+1F36C|U+1F36D|cake|sweetshop|tuck|pastry
+en:4Sweets|4confectionery|candies|candy|U+1F36C|U+1F36D|sweetshop|tuck
 en-AU:Lollies|Candy
-ru:4Кондитерская|кондитерские изделия|сладости|торты|десерты|пирожные
-bg:Сладки|сладкарница|бонбони|торти|сладкиши
+ru:4Кондитерская|кондитерские изделия|сладости
```
Comment (biodranik) [inline data/categories.txt:2156]: be?

**Review by biodranik — APPROVED** [review]

File: data/categories.txt [hunk 3]
```diff
@@ -1075,9 +1075,9 @@ zh-Hans:商店
...
 shop-bakery|shop-pastry|@category_eat|@category_food|@shop
-en:3Bakery|U+1F35E
-ru:3Булочная|3пекарня
-be:3Булачная|3Пякарня
+en:3Bakery|3Pastry|Cake|3Cakes|U+1F35E
+ru:3Булочная|3пекарня|Торт|3Торты|Пирожные
+be:3Булачная|3Пякарня|Торт|3Тарты
 bg:3Пекарна
 ar:مخبز|متجر
 cs:3Pekárna|Pekařství
```
Comment (jimcarst) [inline data/categories.txt:1083]:
```suggestion
cs:3Pekárna|Pekařství
nl:3Bakkerij|Bakker|Gebak|Taart|3Taarten
```

File: data/categories.txt [hunk 4]
```diff
@@ -2151,10 +2151,10 @@ sk:Obchod s výpočtovou technikou
...
-en:4Sweets|4confectionery|candies|candy|U+1F36C|U+1F36D|cake|sweetshop|tuck|pastry
+en:4Sweets|4confectionery|candies|candy|U+1F36C|U+1F36D|sweetshop|tuck
 en-AU:Lollies|Candy
-ru:4Кондитерская|кондитерские изделия|сладости|торты|десерты|пирожные
-bg:Сладки|сладкарница|бонбони|торти|сладкиши
+ru:4Кондитерская|кондитерские изделия|сладости
+bg:Сладки|сладкарница|бонбони|сладкиши
 ar:متجر حلويات|حلويات|حلوى|متجر
 cs:Cukrárna
 da:Slikbutik|slikforretning|konfektureforretning|slik|konfekt
```
Comment (jimcarst) [inline data/categories.txt:2160]:
```suggestion
da:Slikbutik|slikforretning|konfektureforretning|slik|konfekt
nl:Banketbakker|snoepwinkel|snoep|snoepjes|zoetigheden
```

**Pull request merged**